LambdaEngine/Source/Networking/API/ClientRemoteBase.cpp

```diff
@@ -148,6 +148,19 @@ namespace LambdaEngine
 		return true;
 	}
 
+
+	bool ClientRemoteBase::SendReliableBroadcast(NetworkSegment* packet, IPacketListener* listener)
+	{
+		if (!m_pServer->SendReliableBroadcast(this, packet, listener))
+		{
+			LOG_WARNING("[ClientRemoteBase]: Can not broadcast packet");
+			return false;
+		}
+
+		return true;
+	}
+
+
```

Comment:
Multiple spaces above and below the function definition. This should be a single space